webpack/scenes/FlatpakRemotes/FlatpakRemotesActions.js

```diff
@@ -0,0 +1,33 @@
+import { translate as __ } from 'foremanReact/common/I18n';
+import { API_OPERATIONS, post } from 'foremanReact/redux/API';
+import api, { orgId } from '../../services/api';
+import { CREATE_FLATPAK_REMOTES_KEY, EDIT_FLATPAK_REMOTES_KEY }
+  from './FlatpakRemotesConstants';
+import { getResponseErrorMsgs } from '../../utils/helpers';
+
+export const createParamsWithOrg = params => ({
+  organization_id: orgId(),
+  ...params,
+});
+const flatpakCreateSuccessToast = (response) => {
+  const { data: { name } } = response;
+  return __(`Flatpak Remote ${name} created`);
+};
+
+export const flatpakCreateErrorToast = error => getResponseErrorMsgs(error.response);
+export const createFlatpakRemote = params => post({
+  type: API_OPERATIONS.POST,
+  key: CREATE_FLATPAK_REMOTES_KEY,
+  url: api.getApiUrl('/flatpak_remotes'),
+  params: createParamsWithOrg(params),
+  successToast: response => flatpakCreateSuccessToast(response),
+  errorToast: error => flatpakCreateErrorToast(error),
+});
+export const editFlatpakRemotes = params => post({
+  type: API_OPERATIONS.POST,
+  key: EDIT_FLATPAK_REMOTES_KEY,
+  url: api.getApiUrl(`/flatpak_remotes/${params.id}/edit`),
+  params,
+  successToast: response => flatpakCreateSuccessToast(response),
+  errorToast: error => flatpakCreateErrorToast(error),
+});
```

Comment:
```suggestion

```
We can remove this in favor of the action here: https://github.com/Katello/katello/pull/11424/files#diff-07ba9190788a19befb0f7d4e5414d86ce200c6b22ff55e4f5431215b3d68b4a2R17 to avoid too many merge conflicts.